tools/buildsys/src/manifest.rs

```diff
@@ -322,11 +324,44 @@ impl Manifest {
         Ok(kits)
     }
 
+    /// List all external kits needed for the build in the format of "<vendor>-<kit_name>"
+    pub fn external_kit_dependencies<P>(&self, build_dir: P) -> Result<Vec<String>>
```

Comment:
I don't think this function has anything to do with the manifest, does it? It's not using `&self` and seems unrelated to the existence of the manifest file. Likely there should be `::load(path)` function on `ExternalKitMetadataView` and this function should be a member of that object.

So maybe `ExternalKitMetadataView::load(path).list()`.

Anyway I don't think it belongs on `Manifest`